src/overlays/gamestates/ovl_title/z_title.c

```diff
@@ -159,19 +128,50 @@ void ConsoleLogo_Main(GameState* thisx) {
     ConsoleLogo_Calc(this);
     ConsoleLogo_Draw(this);
 
-    if (IS_DEBUG) {
-        ConsoleLogo_PrintBuildInfo(&POLY_OPA_DISP);
-    }
-
     if (this->exit) {
+#if ENABLE_HACKER_DEBUG
+#if BOOT_TO_SCENE
```

Comment:
I'm a bit confused about why this is needed, I assume it's supposed to replace the stuff you removed from ``TitleSetup_SetupTitleScreen``? in this case why not booting to the boot menu for every debug cases and determine if it should show the menu or directly jump to the execution process based on which setting you're using?